src/robotHexa.cpp

```diff
@@ -760,6 +763,23 @@ void RobotHexa :: transfer(ControllerDuty& controller, float duration,int transf
     ROS_INFO_STREAM("Failed to reset odometry");
   }
 
+  // reset filter
+  geometry_msgs::PoseWithCovarianceStamped tmp;
+  tmp.header.stamp = ros::Time::now();
+  tmp.header.frame_id = "/odom";
+
+  tmp.pose.pose.position.x = 0;
+  tmp.pose.pose.position.y = 0;
+  tmp.pose.pose.position.z = 0;
+
+  tmp.pose.pose.orientation.x = 0;
+  tmp.pose.pose.orientation.y = 0;
+  tmp.pose.pose.orientation.z = 0;
+  tmp.pose.pose.orientation.w = 1;
+
+  _reset_filter.publish(tmp);
+  ROS_INFO_STREAM("reset_filter sent");
```

Comment:
Could this message be a little less elusive ? My weak memory would soon forget what this means ;-)